src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -162,8 +162,11 @@ private static JsonObject createLoanContext(
       } else {
         int renewalLimit = loanPolicy.getRenewalLimit();
         int renewalsRemaining = renewalLimit - loan.getRenewalCount();
-        loanContext.put("numberOfRenewalsAllowed", renewalLimit);
-        loanContext.put("numberOfRenewalsRemaining", renewalsRemaining);
+        if (renewalsRemaining < 0) {
```

Comment:
Under what circumstances could this happen?

Reply:
After renewal override.

Reply:
Ok, thanks, this is because a renewal override increments the renewal count?

Reply:
Yes, it does.